ngraph/frontend/onnx/frontend/src/frontend.cpp

```diff
@@ -53,3 +54,25 @@ std::shared_ptr<ngraph::Function> FrontEndONNX::decode(InputModel::Ptr model) co
     NGRAPH_CHECK(model_onnx != nullptr, "Invalid input model");
     return model_onnx->decode();
 }
+
+std::string FrontEndONNX::get_name() const
+{
+    return "onnx";
+}
+
+bool FrontEndONNX::supported_impl(const std::vector<std::shared_ptr<Variant>>& variants) const
+{
+    if (variants.size() > 0 && is_type<VariantWrapper<std::string>>(variants[0]))
+    {
+        const auto path = as_type_ptr<VariantWrapper<std::string>>(variants[0])->get();
+        if (file_util::get_file_ext(path) == ".onnx")
+        {
+            return true;
+        }
+        if (file_util::get_file_ext(path) == ".prototxt")
+        {
+            return true;
+        }
```

Comment:
really? :) we are compiling with protobuf-lite which does not support prototxt files

Reply:
done, handling `prototxt` removed